language-support/ts/codegen/tests/ts/build-and-lint-test/src/__tests__/test.ts

```diff
@@ -26,6 +26,7 @@ const BOB_TOKEN = computeToken(BOB_PARTY);
 const SANDBOX_PORT = 6865;
 const SANDBOX_PORT_FILE = 'sandbox.port';
 const JSON_API_PORT = 7575;
```

Comment:
Hardcoding port numbers somewhat defeats the point of using `--port-file`. We should bind to port `0` and then read the port from the file. Fine to leve it for this PR but let’s change that separately.

Reply:
Tracking in issue https://github.com/digital-asset/daml/issues/5551